test/test_cli_integration.py

```diff
@@ -215,6 +215,19 @@ def test_https_can_list_resources(self):
         self.assertEquals(output,
                           '[\n    "dev:policy:root",\n    "dev:variable:one/password"\n]\n')
 
+    @integration_test
+    def test_https_can_get_whoami(self):
+        self.setup_cli_params({
+            **self.HTTPS_ENV_VARS,
+            **self.HTTPS_CA_BUNDLE_ENV_VAR
+        })
+
+        output = invoke_cli(self, self.cli_auth_params, ['whoami'])
+        response = json.loads(output)
+        self.assertEquals(response.get('account'), 'account')
+        self.assertEquals(response.get('username'), 'user')
+
+
```

Comment:
Extra newline not needed. Also, this test should probably move to the bottom of the file since it's at the bottom of client/api files